timber_modisette/pythonOOP/1OOPbike.py

```diff
@@ -0,0 +1,42 @@
+class Bike(object):
+	def __init__(self, price, max_speed, miles=0):
+		self.price =price
+		self.max_speed =max_speed
+		self.miles =miles
+
+	def displayinfo(self):
+		print "Price:{} Max speed:{} Miles:{}".format(self.price, self.max_speed, self.miles)
+
+	def ride(self,num):
+		print "riding "*num
+		self.miles = self.miles + (10*num) 
+	
```

Comment:
A good idea for your `ride()` function (and similar functions) would be to give `num` a default value of `1`.  That way if you only wanted to ride once you could simply invoke `ride()` vs `ride(1)`.  It would look like this...
``` Python
def ride(self, num=1):
    print "riding " * num
    self.miles = self.miles + (10 * num)
```